build/Targets/Tools.props

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <dotnetRuntimeVersion>2.0.0</dotnetRuntimeVersion>
-    <dotnetSdkVersion>2.1.300-preview2-008324</dotnetSdkVersion>
+    <dotnetRuntimeVersion>2.1.0</dotnetRuntimeVersion>
+    <dotnetSdkVersion>2.1.300</dotnetSdkVersion>
```

Comment:
Have you verified that source build has set this as the LKG yet? Until that is done we can't uptake a CLI version. 

@333fred

Reply:
Where would I look for this?

Reply:
https://github.com/dotnet/source-build/blob/master/DotnetCLIVersion.txt

We cannot exceed the value in that file.

Reply:
Can we add a comment in Tools.props stating this requirement?

Reply:
That's fine by me

Reply:
@eerhardt @dagood Is anything holding back bumping this to RTM?

Reply:
Not that I know of. I just updated corefx to `2.1.300` yesterday - https://github.com/dotnet/corefx/pull/30555

Reply:
Oh, I just read the rest of the thread.  @dseefeld @dleeapho @dagood - it looks like `release/2.1` of source-build is "ahead" of `master`.  What is the plan for keeping the `master` branch up to date?

Reply:
Filed https://github.com/dotnet/source-build/issues/617 to update to 2.1.300, and https://github.com/dotnet/source-build/issues/616 to update to 2.1.301.

I don't think there's a plan to continuously keep `master` up to date. As far as I know there's not currently a place that lists the latest value that we could auto-update from.

Reply:
@dagood @errhard where should we look then for the latest LKG? This file is where I was told to look last time I asked and hence why I'm holding this particular bar. Overall though I'm just trying to ensure we are in sync with the right CLI to keep source build clean. If there is a better place to look for the LKG then we can look there.

Reply:
I think that's the right place to look for the LKG for Roslyn `master`. (Thanks for holding the bar, by the way.) I phrased this pretty badly:

> I don't think there's a plan to continuously keep `master` up to date.

Right now we do want to keep both `master` and `release/2.1` up to date, but we can only do this by hand. (Filed https://github.com/dotnet/source-build/issues/617 to fix `master`.) `release/2.1` right now is the one that gets more attention because we're shipping releases from it. The update to `2.1.300` was a last-minute fix that wasn't ported over to `master`.

What I meant is that there's not currently any plan to be able to do this *automatically/reliably*. (Filed https://github.com/dotnet/source-build/issues/618 to try to get a plan.)